app/apptesting/test_helpers.go

```diff
@@ -0,0 +1,67 @@
+package apptesting
+
+import (
+	"github.com/Stride-Labs/stride/app"
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	minttypes "github.com/cosmos/cosmos-sdk/x/mint/types"
+	"github.com/stretchr/testify/suite"
+	"github.com/tendermint/tendermint/crypto/ed25519"
+	tmtypes "github.com/tendermint/tendermint/proto/tendermint/types"
+)
+
+const Bech32Prefix = "stride"
+
+type AppTestHelper struct {
+	suite.Suite
+
+	App         *app.StrideApp
+	Ctx         sdk.Context
+	QueryHelper *baseapp.QueryServiceTestHelper
+	TestAccs    []sdk.AccAddress
+}
+
+func (s *AppTestHelper) Setup() {
+	config := sdk.GetConfig()
+	config.SetBech32PrefixForAccount(Bech32Prefix, Bech32Prefix+sdk.PrefixPublic)
+
+	checkTx := false
+	s.App = app.InitTestApp(checkTx)
+	s.Ctx = s.App.BaseApp.NewContext(checkTx, tmtypes.Header{Height: 1, ChainID: "stride-1"})
+	s.QueryHelper = &baseapp.QueryServiceTestHelper{
+		GRPCQueryRouter: s.App.GRPCQueryRouter(),
+		Ctx:             s.Ctx,
+	}
+	s.TestAccs = CreateRandomAccounts(3)
+}
+
+func (s *AppTestHelper) FundModuleAccount(moduleName string, amount sdk.Coin) {
+	err := s.App.BankKeeper.MintCoins(s.Ctx, moduleName, sdk.NewCoins(amount))
+	s.Require().NoError(err)
+}
+
+func (s *AppTestHelper) FundAccount(acc sdk.AccAddress, amount sdk.Coin) {
+	amountCoins := sdk.NewCoins(amount)
+	err := s.App.BankKeeper.MintCoins(s.Ctx, minttypes.ModuleName, amountCoins)
+	s.Require().NoError(err)
+	err = s.App.BankKeeper.SendCoinsFromModuleToAccount(s.Ctx, minttypes.ModuleName, acc, amountCoins)
```

Comment:
why minttypes?

Reply:
Great question! It doesn't really matter what module we use here since the funds are immediately moved out of the module account after minting. But since this function is defined at the app level (not module level), I thought it made more sense to use a generic cosmos module instead of stakeibc - and mint seemed suitable. I believe I also saw one of the other repo's using mint

Reply:
Huh, weird. The only slight advantage of using the bank module is that it'll always be initialized, but using the mint module seems fine!

Reply:
I agree, I would prefer to use the bank module too. But hacking around it does make me mildly nervous. I think I'll keep it with the mint module cause I'm pretty confident that's how the other project did it, if I'm remembering correctly